src/plugins/partner_sku_reviews.coffee

```diff
@@ -1370,6 +1406,15 @@ class PartnerSkuReviews
       background: #e8760a;
     }
 
+    /* EXTENDED */
+
+    @media only screen and (max-width: 360px) {
```

Comment:
Media queries do not work in this context because they adopt to the size of the whole viewport while the widget needs to adopt to the size of the container it is put in.

We need a layout that is fluid in any width and we can probably achieve this by using flexbox.